recipes/_carbon_config.rb

```diff
@@ -17,6 +17,14 @@
 # limitations under the License.
 #
 
+directory "conf dir" do
+  path "#{node['graphite']['base_dir']}/conf"
+  owner node['graphite']['user']
+  group node['graphite']['user']
```

Comment:
Thanks, I appreciate that. I mean, you weren't wrong per se... I made a dumb mistake in writing that and it didn't get caught in review. Thanks for jumping on that PR and fixing it, and I'm sorry that I broke it for you.